examples/player/simid_player.html

```diff
@@ -79,7 +64,7 @@
       }
       simidPlayer.stopAd();
       simidPlayer.sendLog("User clicked close ad button on player");
-      setCreativeControlsState(/* controlState= */ DISABLED);
+      simidPlayer.setCreativeControlsState_(/* controlState= */ DISABLED);
```

Comment:
probably also fine to remove the controlState comment now.

Reply:
i spoke to Dan about this, since we are using consts that are really just booleans it is good to explain what the parameters actually are

Reply:
That was more since you were originally just using the literal `true` and `false`.  Now that you've named them, you can remove them if you want (also doesn't hurt to leave it in if you think it adds additional value)

Reply:
Since this function is being called in this file as well as the nonlinear_tester and I think it definitely adds value there since the function isn't defined in that file, then we should stay consistent and it doesn't hurt

Reply:
Using comments is common when you use a primitive type like:

setControlState(false);

A reader would wonder what false meant. So you would add a comment.

However, setControlState(ENABLED); is perfectly readable and understandable on its own.

In cases where you do need to define what a variable is, prefer putting that right into it's name. In this case it would be
const CONTROL_STATE_ENABLED = true;

Your function would then become.
setControlState(CONTROL_STATE_ENABLED);  This would also be readable, but IMO overly verbose.  

And this is right out.
setControlState( /*control state */ CONTROL_STATE_ENABLED);

Hopefully that makes sense, I wish I could link effective java but there essentially a whole chapter on using enums or constants as a way of making code clear and readable without excessive comments.